core/runtime/src/main/java/io/quarkus/runtime/util/EnumerationUtil.java

```diff
@@ -0,0 +1,69 @@
+package io.quarkus.runtime.util;
+
+import java.util.Enumeration;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.Spliterator;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+/**
+ * Transform to "old school" Enumeration from Iterator/Spliterator/Stream
+ */
+public class EnumerationUtil {
```

Comment:
This is probably leftover from previous PR, it will be used in the 2nd (or 3rd) PR -- when I move all of the code from initial (the huge closed one) PR.